portfolio/src/main/webapp/script.js

```diff
@@ -64,16 +64,31 @@ function fetchAndWriteMsg() {
     });
 }
 
-/** Function that fetches a comment from the server and displays it in the comment section. */
+/** Function that fetches a comment from the server and displays it in the comment section. 
+    Automatically only displays 5 comments unless user picks another amount. 
+*/
 function fetchAndDisplayComments() {
     const dataListElement = document.getElementById('comments-container');
-    fetch('/data').then(response => response.json()).then((data) => {
+    fetch('/data?num-comments=5').then(response => response.json()).then((data) => {
         data.forEach((comment) => {
             dataListElement.appendChild(createCommentElement(comment));
         });
     });
 }
 
+/** Function that fetches a specified number of comments from the server and displays it in the comment section 
+    (only called when a value is picked from the dropdown). 
+*/
+function fetchAndDisplayNumComments(num) {
```

Comment:
it looks like this method is the same as the one above (other than clearing out the text in the comments-container). Could you create a separate method called by both so we don't duplicate the logic? 

Also if the first line is just clearing the comments-container, it would be okay if that is called in previous method as well right? If so, one thing you can look into is to see if javascript supports default parameters (I am assuming it does)?

Reply:
Fixed! I just passed in the default parameter to the html call in the body onload function and it works like it did before.